Userland/Libraries/LibDSP/ProcessorParameter.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021, kleines Filmröllchen <malu.bertsch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Music.h"
+#include <AK/Types.h>
+#include <LibCore/Object.h>
+
+namespace LibDSP {
+
+// Processors have modifiable parameters that should be presented to the UI in a uniform way without requiring the
+class ProcessorParameter : public Core::Object {
+    C_OBJECT_ABSTRACT(ProcessorRangeParameter);
+};
+
+namespace Detail {
+
+template<typename ParameterT>
+class ProcessorParameterSingleValue : public ProcessorParameter {
+
+public:
+    ProcessorParameterSingleValue(ParameterT initial_value)
+        : m_value(initial_value)
+    {
+    }
+
+    ~ProcessorParameterSingleValue() requires(IsDestructible<ParameterT>)
+    {
+        m_value.~ParameterT();
+    }
+
+    ParameterT value() const { return m_value; };
+    void set_value(ParameterT value)
```

Comment:
Let's either put both of those methods on one line, or both spread over three lines.